gym_xiangqi/board.py

```diff
@@ -15,4 +15,5 @@ def __init__(self):
     def load_board_image(self):
         filename = "BOARD.png"
         target_file = PATH_TO_BOARD + filename
+        print(target_file)
```

Comment:
is this printing left on purpose here? If so, nvm.

Reply:
Good spot! Removed it.